lib/obp/access/elements/figure.rb

```diff
@@ -0,0 +1,31 @@
+module Obp
+  module Access
+    class Renderer
+      class Elements
+        class Figure < Base
+          def self.classes
+            %w[sts-fig]
+          end
+
+          def content
+            Nokogiri::XML::Builder.new do |xml|
+              xml.fig do
+                xml.label node.at_css(".sts-caption-label").content
+                xml.caption do
+                  xml.title node.at_css(".sts-caption-title").content
+                end
+                xml.graphic("xlink:href": image) # TODO: How to render image?
```

Comment:
@ronaldtse I need to implement image fetching here.

How and where do you want me to store the images? 

I saw that we can use base64 images directly stored in the xml, as binary data. Do you want me to use this?

Reply:
And I saw that we can have many images, titles, captions in a single figure. It seems like a problem, as the specs specify that a figure can contain just a single title, caption or image, not many as follows:

![image](https://github.com/user-attachments/assets/6d0adf95-ba9c-40e4-b221-dd4792bdf904)